psutil/_psutil_bsd.c

```diff
@@ -812,6 +862,7 @@ psutil_users(PyObject *self, PyObject *args) {
 #else
     struct utmpx *utx;
 
+    setutxent();
```

Comment:
No, I have not tested this on FreeBSD.
There is a man page on FreeBSD:
https://www.freebsd.org/cgi/man.cgi?query=setutxent&apropos=0&sektion=0&manpath=FreeBSD+10.2-RELEASE&arch=default&format=html
which doesn't mention that the function is broken.